src/python/qequlacs/gates.py

```diff
@@ -0,0 +1,135 @@
+"""The purpose of this module is to add wrapper/substitution classes
+for gates in Pyquil:
+ * XX - XX=X⊗X
+ * YY - YY=Y⊗Y
+ * ZZ - ZZ=Z⊗Z
+"""
+import numpy as np
+
+from pyquil.quilatom import (
+    Addr,
+    Expression,
+    MemoryReference,
+    MemoryReferenceDesignator,
+    ParameterDesignator,
+    QubitDesignator,
+    unpack_classical_reg,
+    unpack_qubit,
+)
+
+from pyquil.quil import DefGate
+from pyquil.quilatom import Parameter, quil_sin, quil_cos, quil_exp
+from pyquil.quilbase import (
+    AbstractInstruction,
+    Gate
+)
+
+
+def XX(angle: ParameterDesignator, q1: QubitDesignator, q2: QubitDesignator) -> Gate:
```

Comment:
@simonwa7 @dexter2206 – don't you think this should actually go to `z-quantum-core`?

Reply:
https://github.com/zapatacomputing/z-quantum-core/blob/dd6fb59af2b7695cb6a8ad1fe62891e36d73e578/src/python/zquantum/core/circuit/gates/_two_qubit_gates.py#L43 I think it is in the new branch with circuits and gates rework :)

Reply:
@dexter2206 how far are you from merging it to `dev`?